src/sentry/tasks/post_process.py

```diff
@@ -677,6 +688,9 @@ def update_event_groups(event: Event, group_states: Optional[GroupStates] = None
         with sentry_sdk.start_span(op="tasks.post_process_group.fetch_buffered_group_stats"):
             fetch_buffered_group_stats(rebound_group)
 
+        with sentry_sdk.start_span(op="tasks.post_process_group.fetch_buffered_group_last_seen"):
+            fetch_buffered_group_last_seen(event, rebound_group)
+
```

Comment:
Why do we call this in post process group? You mostly need to call this when serializing groups as in issue search